src/core/App.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from "react";
+import { get } from "lodash";
```

Comment:
Avoid bundling the entire lodash library by requiring 

```
import get from "lodash/get";
```

instead

See https://lodash.com/, "Cherry-pick methods for smaller browserify/rollup/webpack bundles."

Reply:
Imported "lodash/get" everywhere where we use "lodash"